Sources/Private/Model/Assets/AssetLibrary.swift

```diff
@@ -18,14 +18,17 @@ final class AssetLibrary: Codable, AnyInitializable, Sendable {
     var imageAssets = [String : ImageAsset]()
     var precompAssets = [String : PrecompAsset]()
 
-    while !container.isAtEnd {
-      let keyContainer = try containerForKeys.nestedContainer(keyedBy: PrecompAsset.CodingKeys.self)
-      if keyContainer.contains(.layers) {
-        let precompAsset = try container.decode(PrecompAsset.self)
+    while
+      !container.isAtEnd,
+      let keyContainer = try? containerForKeys.nestedContainer(keyedBy: PrecompAsset.CodingKeys.self)
```

Comment:
Should we add resilient decoder https://github.com/airbnb/ResilientDecoding in case we need the error at another time?

Reply:
We don't use the `Codable` decoding implementation by default (the dictionary-based implementation is the default in Lottie 4.0+ since it's a lot faster), so no need to invest much in the `Codable` implementation